frontend/src/components/ConnectButton.tsx

```diff
@@ -0,0 +1,23 @@
+// app/components/ConnectButton.tsx
+
+
+import { useMetaMask } from 'metamask-react';
+
+const ConnectButton = () => {
+  const { status, connect, account } = useMetaMask();
+
+  const handleConnect = () => {
+    if (status === 'notConnected') {
+      connect();
+    }
+  };
+
+  return (
+    <div className='flex' onClick={handleConnect}>
+      {status === 'connected' ? 'Metamask Connected' : 'Connect MetaMask'}
```

Comment:
@theGeniusDennis If the status is connected, display the connected account address instead of **Metamask Connected**.
The **useMetamask** hook also provides an address.

Reply:
@theGeniusDennis Is this something that you can get done?

Reply:
Okayy sure. I'll work on that